addons/mail/wizard/mail_compose_message_views.xml

```diff
@@ -25,8 +25,8 @@
                         <field name="email_from"
                             attrs="{'invisible':[('composition_mode', '!=', 'mass_mail')]}"/>
                         <label for="partner_ids" string="Recipients" attrs="{'invisible': ['|', ('is_log', '=', True), ('composition_mode', '!=', 'comment')]}"/>
-                        <div groups="base.group_user" attrs="{'invisible': [('is_log', '=', True)]}">
-                            <span name="document_followers_text" attrs="{'invisible':['|', ('model', '=', False), ('composition_mode', '=', 'mass_mail')]}">Followers of the document and</span>
+                        <div groups="base.group_user" attrs="{'invisible': ['|', ('is_log', '=', True), ('composition_mode', '!=', 'comment')]}">
+                            <span name="document_followers_text" attrs="{'invisible':['|', ('model', '=', False), ('composition_mode', '=', 'mass_mail')]}">Followers of the document and </span>
```

Comment:
useless diff here

Reply:
Hey, the space is intended because in cases where the field is next to that sentence, they just look too close to one another. I could add padding/margin if you'd rather but having 1 character of spacing seems to make sense as it's the continuation of the sentence.